README.md

```
@@ -267,6 +267,7 @@ NIRMAAN/
 
 ---
 
+https://media.giphy.com/media/3o6Zt481isNVuQI1l6/giphy.gif
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)